src/subscription/setup.py

```diff
@@ -29,8 +29,8 @@
 setup(
     name='subscription',
     version=VERSION,
-    description='An Azure CLI Extension to manage Subscription Definitions',
-    long_description='An Azure CLI Extension to manage Subscription Definitions',
+    description='An Azure CLI Extension to manage Subscriptions',
+    long_description='An Azure CLI Extension to manage Subscriptions',
```

Comment:
The description does not conform to https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md

Currently, this is the description we created, "Support for subscription definitions preview.". If you're okay with this, please change the description to that or something else.
This 'guideline' was added as now we list extensions at https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-list

Reply:
Also, if the extension is in preview, follow the instructions at https://github.com/Azure/azure-cli/blob/dev/doc/extensions/metadata.md#azextispreview to mark it as such.

Reply:
Fixed